### DISCUSSION TITLE
load and save data in a file ?

### DISCUSSION THREAD

**Discussion by DjPoke** [nesbox/TIC-80#2021] [Q&A] [answered]
Hello ! Is it actually possible to load/save data in a txt file with lua ? I would like to export some data from programs i'll make, but i don't know how to proceed.

**Comment by DjPoke** [comment, marked as answer]
... or another way like copy data in the clipboard ?

**Reply by joshgoebel** [reply]
You could write it to the cartridge "memory" and then write a program to read the cartridge data off the disk. The format is documented in the wiki.

**Reply by DjPoke** [reply]
Great, thank you !
